content/api/commands/attachfile.md

```diff
@@ -0,0 +1,213 @@
+
+```javascript
+cy.readFile('users.json').then((users => {
+  users[0].username = 'JohnCena'
+})).as('myFile')
+
+cy.get('input[type=file]').attachFile('@myFile')
+```
+
+### Attaching multiple files
+
+```javascript
+cy.get('input[type=file]').attachFile(['file1.json', 'file2.json', 'file3.json'])
+```
+
+This will fail unless the file input has the `multiple` property.
+
+### Attaching a file with custom filename and lastModified
+
+```javascript
+cy.get('input[type=file]').attachFile({
+  contents: 'path/to/file.json',
+  fileName: 'users.json',
+  lastModified: new Date('Feb 18 1989').valueOf()
+})
+```
+
+### Dropping a file on the document
+
+```javascript
+cy.document().attachFile('file.json', { action: 'drag-n-drop' })
+```
+
+## Notes
+
+### Existence
+
+#### Default file existence assertion
+
+When passed a file path, `cy.attachFile()` asserts that the file exists and will fail if it does not exist. It will retry reading the file if it does not initially exist until the file exists or the command times out.
+
+```javascript
+// will fail after the defaultCommandTimeout is reached
+cy.get('input[type=file]').attachFile('does-not-exist.yaml')
+```
+
+### Actionability
+
+#### The element must first reach actionability
+
+`.attachFile()` is an "action command" that follows all the rules
+[defined here](/guides/core-concepts/interacting-with-elements).
+
+## Rules
+
+### Requirements [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Chains-of-Commands)
+
+<List><li>`.attachFile()` requires being chained off a command that yields DOM
+element(s). With the `input` action (default), it further requires a single `input` element with `type="file"`, or a `label` element attached to one.</li><li>If given a path, `cy.attachFile()` requires the file must exist.</li><li>If given an alias, `cy.attachFile()` requires that the subject of the alias must not be `null` or `undefined`</li></List>
+
+### Assertions [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Assertions)
+
+<List><li>`.attachFile()` will automatically wait for the element to reach an
+[actionable state](/guides/core-concepts/interacting-with-elements)</li></List>
+
+### Timeouts [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Timeouts)
+
+<List><li>`.attachFile()` can time out waiting for the element to reach an
+[actionable state](/guides/core-concepts/interacting-with-elements).</li></List>
+
+## Command Log
+
+**_Attach file to input_**
+
+```javascript
+cy.get('.file-input').attachFile(Buffer.from('Hello world'))
+```
+
+The commands above will display in the Command Log as:
+
+<DocsImage src="/img/api/attachfile/attach-file-during-test.png" alt="Command log for attachFile" ></DocsImage>
+
+When clicking on `click` within the command log, the console outputs the
+following:
+
+<DocsImage src="/img/api/click/attach-file-in-console.png" alt="console.log for attachFile" ></DocsImage>
```

Comment:
Did you want this image in the click folder? Will need either an updated path or to pop the png in `/attachfile`

Reply:
Just a copy paste error, fixed.